reference/api/documents.mdx

```diff
@@ -301,21 +384,67 @@ Delete one document based on its unique id.
 
 You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks#get-one-task).
 
+## Delete documents by filter
```

Comment:
Why not just call it `Delete documents` ? It does not exist yet and I suppose we want to promote that route as the default route to delete documents

Reply:
We have an endpoint called "Delete all documents". It might be confusing for the users if we have "Delete documents" and "Delete all documents". "Delete documents by filter" is more specific so sounded like a better option

Reply:
For me `Delete Documents` and `Delete all documents` are clearly distinguishable from each other. But it's your call :)

Reply:
Sorry, I forgot to add this before. We have four delete documents routes:
- Delete all documents
- Delete one document
- Delete documents by filter
- Delete documents by batch

If we didn't have the delete by batch route, "Delete documents" would have worked.